app/assets/stylesheets/application.scss

```diff
@@ -1,4 +1,5 @@
 /*
+ *= require github
```

Comment:
Okay, I saw below... maybe rename this to something more meaningful? I don't know, maybe `tiny_mce_github_theme`, or something like that, so we know it's used for tinyMCE.